marconi-sidechain/doc/API.adoc

```diff
@@ -245,7 +258,7 @@ Taking the chainpoint before ensure that we have consistent infomation across al
 
 ==== Pre-conditions
 
-None as there are no required inputs.
+None as the input parameter is unused.
```

Comment:
Now that I have a closer look at it, isn't both this version and the original one inaccurate? There is indeed a precondition on the parameter, as we don't accept arbitrary values.

Reply:
Good catch. I'll change it. This applies to both `getCurrentSyncedBlock` and `getTargetAddresses`.